cypress/e2e/donor/choose-attorneys-task.cy.js

```diff
@@ -1,6 +1,6 @@
 import {TestEmail} from "../../support/e2e";
 
-describe('Choose attorneys task', () => {
+describe.skip('Choose attorneys task', () => {
```

Comment:
I think it's nice to have some tests that document these flows, because there are too many. But they are slow. So skipping them seems like a sensible middle ground?

Reply:
Yeah good compromise 👍